src/workers/userUpdatedCio.ts

```diff
@@ -0,0 +1,54 @@
+import { getFirstName, ghostUser } from '../common';
+import { TypedWorker } from './worker';
+import { cio, dateToCioTimestamp } from '../cio';
+import { User } from '../entity';
+
+function camelCaseToSnakeCase(
+  obj: Record<string, unknown>,
+): Record<string, unknown> {
+  const snakeObj: Record<string, unknown> = {};
+  for (const key in obj) {
+    snakeObj[key.replace(/[A-Z]/g, (letter) => `_${letter.toLowerCase()}`)] =
+      obj[key];
+  }
+  return snakeObj;
+}
```

Comment:
make this a global util?

Reply:
yes!

Reply:
One edge case with implementation:
```js
camelCaseToSnakeCase({
    SomeProp: 'test'
})
```

outputs:
```
{
    _some_prop: 'test'
}
```

Reply:
Since we already have lodash maybe use `_.snakeCase`?

Reply:
Good idea!